src/main.py

```diff
@@ -164,8 +178,13 @@ def handle_response(text: str) -> str:
 # Message Manager
 async def handle_message(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
     text: str = update.message.text
-    response: str = handle_response(text)
+    response: str
 
+    if feedback:  # Condition to switch to feedback handling mode if set to True
+        await feedback_receive(update, context)  
+        return
+
+    response = handle_response(text)
```

Comment:
**Review: Readability Enhancements Suggested**

The frequency of these conditional checks is expected to increase. It is advisable to leverage Python's match case construct for validating available modes. This will help maintain readability.

```python
# Suggested Refinement Using Match Case Construct
match feedback:
    case True:
        await feedback_receive(update, context)
        return
    case _ :
        response = handle_response(text)
        warning("User:{} :{}-{}".format(update.message.chat.id, text, str(datetime.now())))
        await update.message.reply_text(response)